pkg/images/defaults/flavor.go

```diff
@@ -121,7 +122,8 @@ func DevelopmentBuildImageFlavor() ImageFlavor {
 		ScannerDBSlimImageName: "scanner-db-slim",
 
 		ChartRepo: ChartRepo{
-			URL: "https://charts.stackrox.io",
+			URL:     "https://mirror.openshift.com/pub/rhacs/charts",
+			IconURL: "https://raw.githubusercontent.com/stackrox/stackrox/master/image/templates/helm/shared/assets/Red_Hat-Hat_icon.png",
```

Comment:
[optional] I hope we never rename that branch or remove the icons :) Maybe we should replace `master` with a SHA?

Reply:
[optional] It looks like the string `https://raw.githubusercontent.com/stackrox/stackrox/master/image/templates/helm/shared/assets` may b a good candidate for a `const` as we are using it in many places in several go files.

Reply:
That's a very reasonable idea. Let me merge the PR itself, and then update the URLs in a second PR afterwards.